### PR TITLE
HTML-JS-CSS Capstone Project

### PR DESCRIPTION
**Pull request by lilyfrancis**
This is my capstone project on microverse to show my knowledge on HTML, CSS and Javascript, In this project I build an e-learning conference website; I added the following file.

- HTML File
- CSS File
- Javascript File
- ReadMe File

## The Project presentation video
[Presentation video link](https://www.loom.com/share/52e37f6b5c3946268238168c2157c0f4)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -99,7 +99,7 @@ <h4>Workshop.</h4>
         <div class="program">
           <i class="far fa-handshake fa-2x"></i>
           <h4>E Ignite!</h4>
-          <p>Get opportunities to network with E-learning affliate around the world, also 
+          <p>Get opportunity to network with E-learning affliate around the world, also 
             after the summit. Great Networking Event</p>
         </div>
       </div>
```
Comment (Davidosky007) [inline index.html:105]: * Kindly fix the menu bar as the menu bar on mobile doesn't appear 
* Also please fix the errors in the console i get this error when i scroll down the page 
image

File: index.html [hunk 2]
```diff
@@ -99,7 +99,7 @@ <h4>Workshop.</h4>
         <div class="program">
           <i class="far fa-handshake fa-2x"></i>
           <h4>E Ignite!</h4>
-          <p>Get opportunities to network with E-learning affliate around the world, also 
+          <p>Get opportunity to network with E-learning affliate around the world, also
```
Comment (Davidosky007) [inline index.html:102]: * Kindly fix the footer section of the about page it needs a proper margin between the logo and text 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Hector096 @lilyfrancis 👋, 

Good job so far! 💪 
* Nice video presentation ✔️
* Well written Readme file ✔️ 
*  Good PR ✔️ 
* No linter errors ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @lilyfrancis  🙋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 # Required Changes ♻️ 🚧 
_Check the comments under the review._

- [ ] Kindly resolve this error  from the console `goUp is null`
 
reference |
------------ |
image |

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
## Hi @lilyfrancis 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request closed without merging**